kobo/apps/subsequences/integrations/google/base.py

```diff
@@ -1,3 +1,5 @@
+# coding: utf-8
+from __future__ import annotations
```

Comment:
No need to add `coding: utf-8` anymore since we don't support Python 2 anymore.

Reply:
ok, removed it